uber/tests/email/email_fixtures.py

```diff
@@ -0,0 +1,165 @@
+from uber.tests import *
+from unittest.mock import patch, Mock
+
+
+class EmailTestsConstants:
+    SUBJECT_TO_FIND = 'CoolCon9000: You Need To Know'
+    IDENT_TO_FIND = 'you_are_not_him'
+
+E = EmailTestsConstants
+
+
+@pytest.fixture
+def remove_all_email_categories(monkeypatch):
+    monkeypatch.setattr(AutomatedEmail, 'instances', OrderedDict())
+
+
+@pytest.fixture
+def add_test_email_categories(remove_all_email_categories):
+    AutomatedEmail(
+        model=Attendee,
+        subject='{EVENT_NAME}: You Need To Know',
+        ident='you_are_not_him',
+        template='unrest_in_the_house_of_light.html',
+        filter=lambda a: a.paid == c.NEED_NOT_PAY,
+        when=(),
+        sender="thomas.light@200X.com",
+        extra_data=None,
+        cc="proto@man.com",
+        bcc=None,
+        post_con=False,
+        needs_approval=True,
+    )
+
+
+@pytest.fixture
+def amazon_send_email_mock():
+    """
+    Patch the actual low-level method that actually sends an email out of our system onto the internet.
+    If this is called, you know that an email was really sent by our email subsystem.
+    """
+    with patch.object(AmazonSES, 'sendEmail', return_value=None) as mock:
+        yield mock
```

Comment:
I.......don't know why I didn't do it this way in the first place.  I think I had some issues with monkeypatch originally, but when I tried it just now it works fine.  So, I'll switch both of these over in order to be consistent with the other tests.